cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -530,8 +530,19 @@ class VeloxPlanConverter::WholeStageResIterFirstStage
       std::vector<std::shared_ptr<ConnectorSplit>> connectorSplits;
       connectorSplits.reserve(paths.size());
       for (int idx = 0; idx < paths.size(); idx++) {
+        auto partitionInfo = extractPartitionColumnAndValue(paths[idx]);
+        std::unordered_map<std::string, std::optional<std::string>>
```

Comment:
Sometimes partition key with same column name can take place several times, path is like `a=2012/a=2`, the original propose is not to support this case?

Reply:
When will several partition values exist for a single split, could you tell me more about that?

Reply:
As in https://iceberg.apache.org/docs/latest/spark-ddl/, it has syntax `CREATE TABLE prod.db.sample (
    id bigint,
    data string,
    category string,
    ts timestamp)
USING iceberg
PARTITIONED BY (years(ts), days(ts), category)`
partition by can be the same column.

I reconsider this case, the path will be ts_years=x/ts_days=x，so there is no possible to exists this case